content/docs-v1/contributors.md

```diff
@@ -63,44 +63,50 @@ contributor:
 ```
 
 ```YAML
-title: Introduction
-layout: page
+contributor:
+  - full_name: Kelly Roland
+```
+
+```YAML
 contributor:
   - id: kroland
 ```
 
-### Contributors in the Page Heading
+For most [page types](/docs-v1/pages/#define-page-types), contributors to the page will be automatically listed at the top of the page just under the title. By default, they will appear with their names and, if given, their titles and affiliations (the `name-title` format). You can change this by specifying a different format either globally or on a page-by-page basis.
 
-For most [page types](/docs-v1/pages/#define-page-types), you will notice that contributors to a page will be automatically listed at the top of the page just under the title. By default, they will appear with their names and, if given, their titles and affiliations. You can override this by specifying a new value either on a page-by-page basis or globally.
+| Format       | Display |
+| ------------ | ------ |
+| `name-title` | Name, title, and affiliation |
+| `name`       | Name only |
+| `false`      | Nothing |
 
-On an individual page, in the page YAML:
+The byline format is set globally for the entire publication in the `content/_data/config.yaml` file:
 
 ```yaml
-contributor_byline: name | name-title | false
+bylineFormat: name-title
 ```
 
-For the entire publication, in the `config.yaml` file:
+It can be overridden on individual pages in the page’s YAML:
 
 ```yaml
-bylineFormat: name | name-title | false
+byline_format: false
 ```
-While `name-title` is the default (and will be used if no value is specified), `name` will omit any title or affiliation information, and `false` will remove the contributor listing from the heading of the page altogether.
 
-{{< q-figure id="modern_contributors-page" >}}
+{{< q-figure id="modern_contributors-page-v1" >}}
 
 {{< q-class "box tip" >}}
 
-- If you specify `bylineFormat: false` in the `config.yaml` you can still have names appear on individual pages by specifying either `contributor_byline: name-title` or `contributor_byline: name` on that page.
+- If you specify `bylineFormat: false` in the `config.yaml` you can still have names appear on individual pages by specifying either `byline_format: name-title` or `byline_format: name` on that page.
```

Comment:
Does "on that page" mean in the page YAML?

Reply:
Good question, and the answer is yes. This is clearer I think?:

> ... on an individual page by specifying either `byline_format: name-title` or `byline_format: name` in the YAML for that page.

Reply:
Yes! Looks good.